### ISSUE TITLE
bug: 'Hide album cards' is now showing 'Albums' text

### ISSUE THREAD

**Issue by anyujinie** [inotia00/ReVanced_Extended#334]
### Type

Error at runtime

### Application

Version 18.06.41 non-root 

### Bug description

The hide album cards patch would previously remove any mention of albums in search results. But now it's showing text 'Albums' (Alben in German)

### Steps to reproduce

Look up any artist like IVE

### Relevant log output

```shell
.
```

### Screenshots or videos

![Screenshot_20230225-132623807 (1)](https://user-images.githubusercontent.com/73917917/221356820-9f9bcdcb-2a60-465c-8ba3-0ae8174fd703.png)

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by inotia00** [comment]
can't reproduce
However, this bug may only occur when certain settings are met.

Please upload a full screenshot of your General Layout settings

**Comment by anyujinie** [comment]
![Screenshot_20230225-134227810.png](https://user-images.githubusercontent.com/73917917/221357366-b06d0d4a-1dfd-46b5-a280-e2b39bdda320.png)

**Comment by ILoveOpenSourceApplications** [comment]
I can say the same is there for me too. Attaching my general layout settings as well.
![Screenshot_20230225-191035](https://user-images.githubusercontent.com/117499019/221360037-e8e81eee-647f-4446-838b-ee2a69bc875d.png)

**Comment by inotia00** [comment]
Presumably, this issue seems to reproduce when `Hide breaking news shelf` is off

**Comment by anyujinie** [comment]
> Presumably, this issue seems to reproduce when `Hide breaking news shelf` is off

Yes, I enabled the breaking news shelf and now the albums is gone and shorts is moved a little lower which looks better. I don't know why without that enabled Shorts are at top of results.

Perhaps these two behavior can be fixed in a next release?

**Comment by inotia00** [comment]
Unfortunately, a fix is not possible with my abilities.

someone's contribution is needed

**Comment by inotia00** [comment]
fixed in [revanced-patches-v2.168.6](https://github.com/inotia00/revanced-patches/releases/tag/v2.168.6)
